ooaclld/ooaclld/datatables.py

```diff
@@ -171,19 +171,19 @@ def col_defs(self):
             ]
         if self.language:
             return [
-                LinkCol(
+                IdCol(
                      self, 
                      "Parameter ID", 
                      sTitle="Feature ID", 
                      sClass="left", 
-                     model_col=OOAValue.parameter, 
-                     get_object = lambda i: i.valueset.parameter),
+                     model_col=OOAParameter.id,
+                     get_object=lambda i: i.valueset.parameter),
```

Comment:
We only need to copy the mechanism that is already in place for the feature title. The only difference is that here we want to display the `id` as link label and have search and sort work with `id`. `model_col` arranges for the latter, and `IdCol` is a `LinkCol` that uses the `id` as link label.